Makefile

```
@@ -0,0 +1,146 @@
+# Makefile for APAE Site Comemorativo
+# This file provides convenient commands for development and deployment
+
+.PHONY: help setup dev prod test clean backup restore monitor
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)